docs/content/development/development-setup.md

```diff
@@ -0,0 +1,28 @@
+---
+weight: 2
+---
+
+When developing, it is nice to have a separate environment to test in so that
+you don't have to worry about breaking your system. An easy way to do this is
+by setting up a minikube cluster since it uses `cri-dockerd` by default. You can
+grab the latest version from their repo's releases page:
+
+> You must grab the latest release from their release's page. The version
```

Comment:
"You must grab ..." here contradicts "You can grab ..." in the previous paragraph.

Reply:
We actually had a discussion in another PR and we can direct them to minikube's getting started page for installation. I took out both of these "grabs"